version-compatibility/forkless-upgrade/src/lib.rs

```diff
@@ -19,9 +18,6 @@ mod backward_compatibility;
 #[cfg(test)]
 mod forward_compatibility;
 
-#[cfg(test)]
-mod gas_price_algo_compatibility;
```

Comment:
why was this removed?

Reply:
It doesn't work with v44 because it used v2 metadata. So the choices were keep v36, which isn't ever gonna be used again or remove the test.